Makefile

```diff
@@ -102,8 +102,17 @@ eunit: couch
 
 .PHONY: javascript
 # target: javascript - Run JavaScript test suites or specific ones defined by suites option
-javascript: all share/www/script/test
-	@dev/run -q --with-admin-party-please test/javascript/run $(suites)
+javascript: all
+	@mkdir -p share/www/script/test
+ifeq ($(IN_RELEASE), true)
+	@cp test/javascript/tests/lorem*.txt share/www/test/
+else
+	@mkdir -p src/fauxton/dist/release/test
+	@cp test/javascript/tests/lorem*.txt src/fauxton/dist/release/test/
+endif
+	@ulimit -n 10240
```

Comment:
```
[couchdb@freebsd ~]$ ulimit -n 1024
-bash: ulimit: open files: cannot modify limit: Operation not permitted
```

That's strange that we can hit emfile here, while with the same strategy for eunit tests we don't. Do we cleanup databases after the tests? Or we just have fd leak? That's important.